ahorro_combustible/utils.py

```diff
@@ -0,0 +1,61 @@
+from typing import Dict, Tuple, Sequence
+
+
+def count_brands(gas_stations) -> Sequence[Tuple[int, str]]:
+    '''Count all the brands in the `gas_stations` input. It can be considered that
+    the ``Rotulo`` entry contains the brand name of the gas station.
+
+    Parameters
+    ----------
+    gas_stations : dict
+        A dictionary that contains at least the equivalent structure to the
+        following example.
+
+        { 'ListaEESSPrecio': [ { 'Rótulo': 'ES DULANTZI REPSOL' }, ... ]}
+
+    Returns
+    -------
+    Sequence[Tuple[int, str]]
+        A sorted sequence of tuples being the first part of each tuple the count
+        number of the second part, the brand name.
+
+        The items will be sorted by the count number (highest, first) and the
+        brand name.
+
+    Examples
+    --------
+
+    >>> count_brands({'ListaEESSPrecio': [ {'Rótulo': 'A' },
+    ...                                    {'Rótulo': 'B' },
+    ...                                    {'Rótulo': 'A' }]})
+    [(2, 'A'), (1, 'B')]
+
+    >>> count_brands({'ListaEESSPrecio': [ {'Rótulo': 'A' },
+    ...                                    {'Rótulo': 'C' },
+    ...                                    {'Rótulo': 'C' },
+    ...                                    {'Rótulo': 'B' }]})
+    [(2, 'C'), (1, 'A'), (1, 'B')]
```

Comment:
This test is wrong. Could you replace it by the following suggestion?

```suggestion
    [(2, 'C'), (1, 'B'), (1, 'A')]
```